actors/miner/tests/util.rs

```diff
@@ -8,3 +27,197 @@ pub fn new_bls_addr(s: u8) -> Address {
     rng.fill_bytes(&mut key);
     Address::new_bls(&key).unwrap()
 }
+
+#[allow(dead_code)]
```

Comment:
we can probably remove it now, i wrote it before the test and rust complained.

Reply:
removed it, no complaints from rust.

Reply:
but we have to keep it for the impl block, if i remove it there as well rust complains.... which is kind of braindead because the fn's are exported and the compiler should be smart enough to see that exported funs are *not* dead code; sigh.